src/map/zone_entities.cpp

```diff
@@ -139,7 +139,7 @@ void CZoneEntities::InsertPET(CBaseEntity* PPet)
             }
             targid++;
         }
-        if (targid >= 0x800)
+        if (targid >= 0x780)
```

Comment:
did not realize we had raw dawg offsets. are these party slots or what?

Reply:
oh its the offset within zone for entity type. probably ought to have these written down someplace.

Reply:
Will open up an issue with all the information about this, and the proposed next steps. But I want to combine that work with some other architectural changes, so I'll leave it in this half-way step for now.